.infrastructure/bootstrap.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
```

Positive: Using `set -euo pipefail` is a good practice (line 3). It causes the script to fail early on errors which is appropriate for a bootstrap script. Keep this. Also the ordering (namespace -> PV/PVC -> config/secret -> statefulset -> deployment) is reasonable once the missing files/names are corrected.